wandb/proto/wandb_internal.proto

```diff
@@ -1361,3 +1363,36 @@ message JobInputRequest {
   repeated JobInputPath exclude_paths = 3;
   string input_schema = 4;
 }
+
+// ServerFeatures related protobufs
+
+enum ServerFeature {
+  LARGE_FILENAMES = 0;
+  ARTIFACT_TAGS = 1;
+  CLIENT_IDS = 2;
+}
+
+/**
+ * ServerFeatureRequest: request to the backend server for the features it supports.
+ * including the details of the feature provided by the server.
+ */
+message ServerFeatureRequest {
+  ServerFeature feature = 1;
+  _RequestInfo _info = 200;
+}
+
+/**
+ * ServerFeatureResponse: response from the backend server for the features it
+ * supports. including the details of the feature provided by the server.
+ */
+message ServerFeatureResponse {
+  ServerFeatureItem feature = 1;
+}
+
+/**
+ * ServerFeatureItem: details of a feature provided by the server.
+ */
+message ServerFeatureItem {
+  string name = 1;
```

Comment:
What's the relationship between the `name` field and the ServerFeature enum?

Reply:
Could this be ServerFeature instead of a string?

Reply:
> Could this be ServerFeature instead of a string

This is a great suggestion!